README.rst

```diff
@@ -91,6 +91,7 @@ Related Projects
 ------------------------------------------------------------------------
 
 - asyncache_: Helpers to use cachetools_ with asyncio.
+- cachetools-async_: Helpers to use cachetools_ with asyncio (more actively maintained than asyncache_).
```

Comment:
Personally, I'd drop the comment ("more actively maintained...") - this may change in the future, and I expect readers to be able to decide for themselves what suites their needs best, so I wouldn't make any judgements here,